core/commands/swarm.go

```diff
@@ -447,6 +463,51 @@ func (ci connInfos) Swap(i, j int) {
 	ci.Peers[i], ci.Peers[j] = ci.Peers[j], ci.Peers[i]
 }
 
+func (ci *connInfo) identifyPeer(ps pstore.Peerstore, p peer.ID) (*IdOutput, error) {
+	info := new(IdOutput)
+	keyEnc, err := ke.KeyEncoderFromString(ke.OptionIPNSBase.Name())
+	if err != nil {
+		return nil, err
+	}
+	info.ID = keyEnc.FormatID(p)
+
+	if pk := ps.PubKey(p); pk != nil {
+		pkb, err := ic.MarshalPublicKey(pk)
+		if err != nil {
+			return nil, err
+		}
+		info.PublicKey = base64.StdEncoding.EncodeToString(pkb)
+	}
```

Comment:
We don't use base64 as-is over here, we use https://pkg.go.dev/github.com/multiformats/go-multibase (tl;dr: the same thing but adds a prefix which allows consumers to know what base was used). :slightly_smiling_face: 
Also I would be surprised if this usefull, I would expect everyone to just care about the peerid, RSA are the only peer id in use which does not embed the public key already.

Reply:
If it is probably not useful, do you thing it's better for me to remove the public keys from the output? I followed the interface suggested in the feature request, I'll change the encoding to multibase for now

Reply:
Nvm leave it as-is, I guess they want base64 to avoid having to multibase in JS which is a bit anoying.
That is good thx.